scripts/inputs/build_inputs.py

```diff
@@ -165,9 +165,32 @@ def process_directory(input_dict, template_dir, target_directory):
             process_file(input_dict, subdir, file, target_subdir)
 
 
+def transpose_tsv(input_file, output_file):
+    with open(input_file) as f:
+        # read lines from the file and remove trailing whitespace
+        lines = [line.rstrip('\n') for line in f.readlines()]
+
+    # group the lines by their column number
+    groups = {}
+    for line in lines:
+        if line:
+            columns = line.split('\t')
+            for i, column in enumerate(columns):
+                if i not in groups:
+                    groups[i] = []
+                groups[i].append(column)
+    # write the transposed data to a new file
+    with open(input_file, 'w') as f:
+        for i in sorted(groups.keys()):
+            f.write('\t'.join(groups[i]) + '\n')
+
+
 def process_file(input_dict, template_subdir, template_file, target_subdir):
+    # transpose files that starts with workspace for Terra processing
+    if template_file.startswith("workspace"):
```

Comment:
I'd move this processing to after we check for the `.tmpl` extension

Reply:
Thank you for your feed back, this request has been addressed [here](https://github.com/broadinstitute/gatk-sv/pull/523/commits/aabccf74c38db6c85598567a8ecb399cfdd96f7e).